libs/core/langchain_core/documents/copy_transformer.py

```diff
@@ -0,0 +1,29 @@
+import copy
+from typing import Any, AsyncIterator, Iterator
+
+# Note: Import directly from langchain_core is not stable and generate some errors
+from langchain_core.documents import Document
+from langchain_core.documents.runnable_document_transformer import (
+    RunnableGeneratorDocumentTransformer,
+)
+
+
+class CopyDocumentTransformer(RunnableGeneratorDocumentTransformer):
```

Comment:
For the moment, I've placed this class here, so as not to create a langchain_core.document_transformers (as in langchain). This can be changed.